recipes/ipython-sql/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.3.6" %}
+{% set md5 = "d4feb00ac5806d7640b2545a43974766" %}
+
+package:
+  name: ipython-sql
+  version: {{ version }}
+
+source:
+  fn: ipython-sql-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipython-sql/ipython-sql-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+skip:
+  true # [py2k]
```

Comment:
So this should be in the `build` section, it should be one line, and the selector should have at least 2 spaces before it.